HTML/index.html

```diff
@@ -13,31 +13,32 @@
   <!-- Hero Section Start -->
 
```

Comment:
Nice work so far 🙂🙂 , but it's always recommended to not leave comments in production codes, please try to remove them before you deploy your project.